api/v340/launch/serializers.py

```diff
@@ -124,18 +124,19 @@ class Meta:
 
 
 class AgencySerializerDetailedForLaunches(QueryFieldsMixin, serializers.HyperlinkedModelSerializer):
-
     class Meta:
         model = Agency
         fields = ('id', 'url', 'name', 'featured', 'type', 'country_code', 'abbrev', 'description', 'administrator',
-                  'founding_year', 'launchers', 'spacecraft', 'launch_library_url', 'successful_launches',
+                  'founding_year', 'launchers', 'spacecraft', 'launch_library_url', 'consecutive_successful_landings',
+                  'successful_landings', 'failed_landings', 'attempted_landings', 'successful_launches',
                   'failed_launches', 'pending_launches', 'info_url', 'wiki_url', 'logo_url', 'image_url', 'nation_url',)
 
 
 class RocketDetailedSerializer(serializers.ModelSerializer):
     configuration = LauncherConfigDetailSerializer(read_only=True, many=False)
     launcher_stage = FirstStageSerializer(read_only=True, many=True, source='firststage')
-    spacecraft_stage = SpacecraftFlightDetailedSerializerForLaunch(read_only=True, many=False, source='spacecraftflight')
+    spacecraft_stage = SpacecraftFlightDetailedSerializerForLaunch(read_only=True, many=False,
+                                                                   source='spacecraftflight')
```

Comment:
Whats up with the formatting here?

Reply:
IntelliJ's auto-formatting doing weird things I suppose.